Alert/schema.json

```diff
@@ -0,0 +1,175 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Alert/schema.json",
+  "title": "Alert data model JSON Schema",
+  "description": "An alert generated by a user or device in a givel location",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons"
+    },
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+      "properties": {
+        "dataTime": {
+          "oneOf": [
+            {
+              "type": "string",
+              "format": "date-time"
+            },
+            {
+              "type": "object",
+              "properties": {
+                "start": {
+                  "type": "string",
+                  "format": "date-time"
+                },
+                "end": {
+                  "type": "string",
+                  "format": "date-time"
+                }
+              }
+            }
+          ]
+        },
+        "description": {
+          "type": "string"
+        },
+        "severity": {
+          "type": "string"
+        },
+        "severityIndex": {
+          "type": "number"
+        },
+        "alertType": {
+          "type": "string",
+          "enum": [
+            "TrafficJam",
+            "CarAccident",
+            "WeatherCondition",
+            "Pollutations",
+            "Pollen",
+            "AsthmaAttacks"
+          ]
+        },
+        "eventObserved": {
+          "type": "string"
+        },
+        "refUser": {
+          "type": "string",
+          "format": "uri"
+        },
+        "refDevice": {
+          "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Alert"
+          ],
+          "description": "NGSI Entity type"
+        }
+      }
+    }
+  ],
+  "oneOf": [
+    {
```

Comment:
Maintenance-wise I don't know if it is a good idea to describe here all the possible combinations ....

Reply:
no we have only 2 combinations, should be ok

Reply:
hello, can we add a new text of subcategory in alert Data model. for example high temperature?o something like SOS Button?